cmd/executor/testdata/Dockerfile.trivial

```diff
@@ -0,0 +1,2 @@
+FROM ubuntu
```

Comment:
this is a little bit odd. all our tests and test data is in 
integration/
Would it possible to move these file in integration ?

Reply:
Done